templates/zerver/api/roles-and-permissions.md

```diff
@@ -0,0 +1,85 @@
+# Roles and permissions
+
+Zulip offers [several levels of permissions based on user
+organization-level roles](/help/roles-and-permissions).
```

Comment:
Probably easier to parse if "user organization-level roles" is unpacked: e.g. "a user's role in a Zulip organization".

Reply:
I'd also shrink the link to just be on "user's role", not the whole sentence.